images/fcos-bastion-image/root/usr/bin/hpsol

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+if [ "${#}" -lt 1 ]; then
+    echo "Usage: ${0} <host_id> <command>"
+    exit 1
+fi
+
+host_id="${1}"
+command="${2}"
+shift
+
+host_obj=$(sed '1s/^#//; 2,${/^#/d; /^$/d}' "/etc/hosts_pool_inventory" | yq -p csv '.[] | select(.bmc_address == "*'".${host_id}"'*")')
+
+user=$(echo "${host_obj}" | yq -r '.bmc_user')
+password=$(echo "${host_obj}" | yq -r '.bmc_pass')
+address=$(echo "${host_obj}" | yq -r '.bmc_address')
+echo -n "${address}: "
+p=$(mktemp)
+echo "${password}" | tee $p
```

Comment:
this was in the temporary script for testing/manual debugging but, do we need to show the password every time we execute this command in the CI? 
```suggestion
echo "${password}" > $p
```